### DISCUSSION TITLE
What is releng?

### DISCUSSION THREAD

**Discussion by SteveSchafer-Innovent** [eclipse-birt/birt#980] [Q&A]
In trying to track down some scripts in the birt build, I see they are called in some scripts in org.eclipse.birt.releng.birtbuilder.  What is that, and where and when are its scripts called?

**Comment by wimjongman** [comment]
That should be defined in a modules section in one of our pom.xml's

**Reply by wimjongman** [reply]
https://github.com/eclipse/birt/search?q=releng.birtbuilder

**Reply by SteveSchafer-Innovent** [reply]
In fact, I can't find it in any modules section.  If I search all pom.xml files for "releng" I find 7 ant scripts where it''s copying html files from releng.birtbuilder.extras and one ant script where it's copying files from releng.birtbuilder/eclipse/buildConfigs/birt.tests/test.scripts.

**Reply by wimjongman** [reply]
I'm not sure if that is required. There could be a maven trigger that I am unaware of.

However, I don't see it mentioned in the build either: e.g. https://github.com/eclipse/birt/pull/939/checks
